pkg/errs/errno.go

```diff
@@ -15,38 +15,38 @@ package errs
 
 import "github.com/pingcap/errors"
 
+// The internal error which is generated in PD project.
 // tso errors
 var (
 	ErrInvalidTimestamp    = errors.Normalize("invalid timestamp", errors.RFCCodeText("PD:tso:ErrInvalidTimestamp"))
 	ErrLogicOverflow       = errors.Normalize("logic part overflow", errors.RFCCodeText("PD:tso:ErrLogicOverflow"))
 	ErrIncorrectSystemTime = errors.Normalize("incorrect system time", errors.RFCCodeText("PD:tso:ErrIncorrectSystemTime"))
 )
 
-// adapter errors
+// member errors
```

Comment:
On what basis do we rank these errors?

Reply:
IMO, we just need to distinguish the errors generated in PD internal or in the third-party package. There is no rank concept.